client/cli/ok.py

```diff
@@ -92,6 +93,12 @@ def parse_input(command_input=None):
 
 def main():
     """Run all relevant aspects of ok.py."""
+
+    #Checking user's Python bit version
+    bit_v = (8 * struct.calcsize("P"))
+    if (bit_v == 32):
+        print("32 bit Python is not supported. Please install the 64 bit version.")
```

Comment:
Following our discussion, looks like this PR should just log the bit version regardless of whether or not it's 32 bit, and not print a message to the user. So effectively:

```python
log.setLevel(...)
bit_v = 8 * struct.calcsize("P")
log.debug("Python bit version: {}".format(bit_v))
```

@Sumukh do you intend for the second step of your proposal to be done in this PR?